api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -132,8 +136,11 @@ public GenomicExtractionService(
                       genomicExtractionMapper.convertWorkflowStatus(
                           // Extraction submissions should only have one workflow.
                           firecloudSubmission.getWorkflows().get(0).getStatus());
-                  dbSubmission.setTerraStatusEnum(status);
-                  if (status != TerraJobStatus.RUNNING) {
+                  if (dbSubmission.getTerraStatusEnum() != TerraJobStatus.ABORTING
```

Comment:
if status is == ABORTED it will always not be equal to ABORTING. What's the purpose of this conditional on the setter?

Reply:
I don't remember. It's possible that Friday Me hated Tuesday Me and wanted me to fail at all my life's endeavors.

Reply:
Ok, the if not aborting conditional is because of me setting it to aborting in L314. The rest of the conditional is just incorrect. I'll pull out the updating of the status and let there be no feedback.